pallets/staking-rewards/src/mock.rs

```diff
@@ -331,8 +332,8 @@ impl pallet_staking::Config for Test {
     type MaxNominations = MaxNominations;
     type Currency = Balances;
     type UnixTime = Timestamp;
-    type CurrencyBalance = u128;
-    type CurrencyToVote = U128CurrencyToVote;
+    type CurrencyBalance = <Self as pallet_balances::Config>::Balance;
```

Comment:
BalanceOf

Reply:
This type comes from pallet-staking config. We can't change this

Reply:
I guess breathx meant `type CurrencyBalance = BalanceOf<Self>`

Reply:
It won't work like that

Reply:
Can't see any reason for this not working, but ok, it doesnt matter in general